app/forms/placement_form.rb

```diff
@@ -112,6 +111,16 @@ def save_and_return_invalid_data!
     save_or_stash && invalid_data
   end
 
+  def school_valid
+    if school_id.blank? && (name.blank? && urn.blank? && postcode.blank?)
+      errors.add(:school, :blank)
+    end
+  end
+
+  def open_details?
```

Comment:
sorry to be ignant: what does `open_details` refer to?

Reply:
It's used here

https://github.com/DFE-Digital/register-trainee-teachers/blob/77f60421b4b9d5257e5e0bfd72137a119ae4ea6d/app/views/trainees/placements/_form.html.erb#L23

Reply:
Discussed IRL